lib/version.rb

```diff
@@ -1,4 +1,4 @@
 module DangerSwiftlint
-  VERSION = "0.16.1".freeze
+  VERSION = "0.4.0".freeze
```

Comment:
For anyone else following along, this is fine because 0.16.1 was never released to RubyGems.org so we're fine to "go backward" in semver.